config.example.json

```diff
@@ -5,6 +5,7 @@
     "clusterBuild": "",
     "nodePort": 4000,
     "id": "Bot ID",
+    "ownerID": "Bot Owner ID",
```

Comment:
As discussed via DM, this should be removed since we have the `maintainers` config option